### PR TITLE
docs: Add contributing instructions

### PR DESCRIPTION
**Pull request by jimmidyson**
- docs: Add contributing instructions
- feat: Fix up post bundle archetype

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,105 @@
 
 ![D2iQ Engineering Blog license](https://img.shields.io/github/license/mesosphere/d2iq-engineering-blog)
 
+This blog is brought to you by the engineers of [D2iQ][d2iq]. This is a place for D2iQ engineers to share our knowledge
```
Comment (cmastr) [inline README.md:5]: limiting this to engineers really reduces the authors who can post here. If someone like an SA or SE wanted to write an article, would they be allowed?

Comment (jimmidyson) [reply README.md:5]: Good point - any suggestions?

Comment (jimmidyson) [reply README.md:5]: Everyone's allowed!

Comment (jimmidyson) [reply README.md:5]: I'm going to merge this now and we can review/improve later.

**Pull request merged**